multer/index.js

```diff
@@ -0,0 +1,74 @@
+const express= require('express');
+const fs = require('fs');
+
+const multer =require('multer');
+
+const app= express();
+  
+const fileStorage=multer({
+    storage: multer.diskStorage({
+        destination:(req, file ,cb)=>{
+            cb(null ,'upload')
+    
+        },
+        filename:(req,file,cb)=>{
+            cb(null,`file${file.fieldname}`)
+        }
+    })
+}).any();
+
+
+const upload= multer({storage:fileStorage});
+
+// upload files
+app.post("/single",fileStorage,(req,res)=>{
+    res.send("file is upload");
+})
+
+app.get("/merge",(req,res)=>{
+    // res.send("file is upload");
+    const resultFileName =`./backup/RES___${Date.now()}.txt`;
+
+    const file1 = './upload/file1';
+    const file2 = './upload/file2';
+    //const des = `./backup/`;
+
+
+    function mergeFiles(file1, file2, des) {
```

Comment:
Try to avoid add functions in the api method. For good readability and to reuse the function always create the function outside.